apps/duality-social-angular/src/core/services/auth.service.ts

```diff
@@ -88,7 +99,7 @@ export class AuthenticationService {
 
   getAccessToken(): Promise<string> {
     return this.msalClient.acquireTokenSilent({
-      scopes: ['User.Read'],
+      scopes: ['User.Read', 'api://25989269-b717-4761-8498-f83e3bfc0754/api.openai'],
```

Comment:
This is a brand new scope I created for the OpenAI API, but I'm not sure if I should be using it here, anywhere else, or what